WooCommerce/src/main/kotlin/com/woocommerce/android/ui/compose/component/Toolbar.kt

```diff
@@ -140,19 +148,25 @@ fun Toolbar(
     )
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun Toolbar(
     modifier: Modifier = Modifier,
     title: @Composable () -> Unit,
     onNavigationButtonClick: (() -> Unit)? = null,
     navigationIcon: ImageVector? = null,
     navigationIconContentDescription: String = stringResource(id = string.back),
-    windowInsets: WindowInsets = WindowInsets(0),
+    windowInsets: WindowInsets = TopAppBarDefaults.windowInsets,
     actions: @Composable RowScope.() -> Unit = {}
 ) {
     TopAppBar(
         windowInsets = windowInsets,
-        backgroundColor = colorResource(id = R.color.color_toolbar),
+        colors = TopAppBarDefaults.topAppBarColors(
+            containerColor = colorResource(id = R.color.color_toolbar),
+            titleContentColor = MaterialTheme.colorScheme.onSurface,
+            navigationIconContentColor = MaterialTheme.colorScheme.onSurface,
+            actionIconContentColor = MaterialTheme.colorScheme.onSurface
```

Comment:
We use primary color for Toolbar actions (you can confirm it in other app screens)
```suggestion
            actionIconContentColor = MaterialTheme.colorScheme.primary
```

Reply:
Good catch!

1f1aaa753919f0363e6bce084ff977e79db26fbc